### PR TITLE
Remove "force" field from the queue Jobs

### PR DESCRIPTION
**Pull request by severo**
Now that all the jobs are created by the DatasetState.backfill() method, we don't want to skip jobs.

Another PR will remove the "skip job" mechanism

### CODE REVIEW

File: jobs/mongodb_migration/tests/migrations/test_20230511100700_queue_delete_indexes_with_force.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+        migration = MigrationQueueDeleteIndexesWithForce(
+            version="20230511100700",
+            description="remove indexes with field 'force'",
+        )
+        migration.up()
+
+        assert (
+            len(get_index_names(db[QUEUE_COLLECTION_JOBS].index_information(), "force")) == 0
+        )  # Ensure the TTL index exists  # Ensure 0 records with old type
```
Comment (AndreaFrancis) [inline jobs/mongodb_migration/tests/migrations/test_20230511100700_queue_delete_indexes_with_force.py:36]: Same

File: services/worker/src/worker/job_runner.py [hunk 2]
```diff
@@ -298,8 +298,8 @@ class JobRunner(ABC):
 
     Args:
         job_info (:obj:`JobInfo`):
-            The job to process. It contains the job_id, the job type, the dataset, the config, the split
-            the force flag, and the priority level.
+            The job to process. It contains the job_id, the job type, the dataset, the config, the split,
```
Comment (AndreaFrancis) [inline services/worker/src/worker/job_runner.py:301]: I think all these changes should go now to job_manager.py file

File: jobs/mongodb_migration/tests/migrations/test_20230511100700_queue_delete_indexes_with_force.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+def test_queue_delete_indexes_with_force(mongo_host: str) -> None:
+    with MongoResource(database="test_queue_delete_indexes_with_force", host=mongo_host, mongoengine_alias="queue"):
+        Job(type="test", dataset="test", unicity_id="test", namespace="test", created_at=get_datetime()).save()
+        db = get_db(QUEUE_MONGOENGINE_ALIAS)
+        db[QUEUE_COLLECTION_JOBS].create_index(field_name)
+        db[QUEUE_COLLECTION_JOBS].create_index([(field_name, 1), ("type", 1)])
+        db[QUEUE_COLLECTION_JOBS].create_index([("type", 1), (field_name, 1)])
+        assert (
+            len(get_index_names(db[QUEUE_COLLECTION_JOBS].index_information(), "force")) == 3
+        )  # Ensure the TTL index exists
```
[the remark this answers is not in the document]
Comment (severo) [reply jobs/mongodb_migration/tests/migrations/test_20230511100700_queue_delete_indexes_with_force.py]: fixed

File: jobs/mongodb_migration/tests/migrations/test_20230511100700_queue_delete_indexes_with_force.py (hunk 1 above)
Comment (severo) [reply jobs/mongodb_migration/tests/migrations/test_20230511100700_queue_delete_indexes_with_force.py]: fixed

File: services/worker/src/worker/job_runner.py (hunk 2 above)
Comment (severo) [reply services/worker/src/worker/job_runner.py]: yes, the rebase was big :)

**Review by AndreaFrancis — APPROVED** [review]
Nice!

**Pull request merged**